README.md

```diff
@@ -24,17 +24,17 @@ The link is shortened version of https://raw.githubusercontent.com/CleanMachine1
 
 ### Manual install
 
-__You need to have **git** and **make** installed to install__
+__You need to have **git** and **make** installed to download the files and compile easier!__
```

Comment:
```suggestion
__To download and compile DollarSkip you need **git** (to download the files) and **make** (to make compilation easier)__
if you don't want to use git and make, you can download the zip from [here](https://github.com/CleanMachine1/DollarSkip/archive/refs/heads/master.zip) and compile `dollarskip.c` with `gcc dollarskip.c -o $`, then move `$` to `/usr/bin`.
```

Reply:
I personally don't like this however it could be edited.

I found trouble with making the binary from gcc called $, instead I called it temp so it could be moved easier without complications.

I will add my suggestions a little later

Reply:
it can be:

if you don't want to use git and make, you can download the zip from [here](https://github.com/CleanMachine1/DollarSkip/archive/refs/heads/master.zip) and compile `dollarskip.c` with `gcc dollarskip.c -o temp`, then move `temp` to `/usr/bin/$` with `sudo mv temp /usr/bin/\$`.

Reply:
That will do however I still don't like the appearance of it physically. So I will put it in a summary.

Reply:
```suggestion


<details>
<summary><b>To install DollarSkip without Git or Make</b></summary>


If you don't want to use Git and Make, you can download the zip from [here](https://github.com/CleanMachine1/DollarSkip/archive/refs/heads/master.zip) and compile `dollarskip.c` with `gcc dollarskip.c -o temp`, then move `temp` to `/usr/bin/$` with `sudo mv temp /usr/bin/\$`.
</details>


```